uc-crux-llvm/test/programs/linked_list_sum.c

```diff
@@ -9,5 +9,6 @@ int linked_list_sum(struct list *l) {
     total += next->head;
     next = next->tail;
   }
+  return total;
```

Comment:
As the commit message of 2725066f8b25211c3e92e9b29946c55c91d9e1cc explains, I needed to make this change in order to prevent Clang 12 from optimizing the entirety of the `linked_list_sum` function to just `ret undef`. I think this is an OK thing to do, but please double-check me @langston-barrett.

Reply:
Yep, that looks like it was just a mistake on my part when creating the test. Thanks for the fix!